packages/fxa-auth-server/lib/senders/emails/templates/inactiveAccountSecondWarning/index.mjml

```diff
@@ -0,0 +1,30 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="inactiveAccountSecondWarning-title">Your Mozilla account and data will be deleted in 7 days</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-sub-body">
+      <span data-l10n-id="inactiveAccountSecondWarning-account-description">Your Mozilla account is used to access free privacy and browsing products like Firefox Sync, Monitor, Relay, and MDN.</span>
```

Comment:
The products listed in this string were updated in Figma.
```suggestion
      <span data-l10n-id="inactiveAccountSecondWarning-account-description">Your Mozilla account is used to access free privacy and browsing products like Firefox sync, Mozilla Monitor, Firefox Relay, and MDN.</span>
```

Reply:
@vpomerleau I thought l10n recommended we don't capitalize Sync? Am I misremembering?